metadata/metadata.go

```diff
@@ -31,12 +31,13 @@ import (
 const (
 	defaultMetadataURL = "http://169.254.169.254/computeMetadata/v1/"
 	defaultEtag        = "NONE"
-	metadataRecursive  = "/?recursive=true&alt=json"
-	metadataHang       = "&wait_for_change=true&timeout_sec=60"
+	defaultTimeout     = 60
 )
 
 var (
-	defaultTimeout = 70 * time.Second
+	// we backoff until 10s
+	backoffDuration = 100 * time.Millisecond
+	backoffAttempts = 100
```

Comment:
is this arbitrary number? 100 feels like a very big number for retries

Reply:
Totally arbitrary, I don't even intend to say these numbers make sense but just a small detail to add, the duration is in milliseconds.